typescript/system.ts

```diff
@@ -0,0 +1,13 @@
+export interface System {
+  formsBuilderAI: {
+    systemPrompt: string
+    previousSystemPrompts: Array<{
+      systemPrompt: string
+      archivedAt: string
+    }>
+    temperature: number
+    topP: number
+    topK: number
+    maxTokensPerRequest: number
```

Comment:
Will this account for request/input and response/output tokens?

Reply:
They don't appear to have a property for input token limit, which leads me to believe its something your supposed to handle yourself API side.